File: src/main/java/waterpower/client/render/IIconRegister.java
package waterpower.client.render;

import net.minecraft.client.renderer.texture.TextureMap;
import net.minecraftforge.fml.relauncher.Side;
import net.minecraftforge.fml.relauncher.SideOnly;

public interface IIconRegister {

	@SideOnly(Side.CLIENT)
    void registerIcons(TextureMap textureMap);

}

File: src/main/kotlin/waterpower/client/render/IIconRegister.kt
/**
 * Copyright (c) Huang Yuhui, 2017
 *
 * "WaterPower" is distributed under the terms of the Minecraft Mod Public
 * License 1.0, or MMPL. Please check the contents of the license located in
 * http://www.mod-buildcraft.com/MMPL-1.0.txt
 */
package waterpower.client.render

import net.minecraft.client.renderer.texture.TextureMap
import net.minecraftforge.fml.relauncher.Side
import net.minecraftforge.fml.relauncher.SideOnly

interface IIconRegister {
    @SideOnly(Side.CLIENT)
    fun registerIcons(textureMap: TextureMap)
}